main/taskd/APKBUILD

```diff
@@ -35,7 +35,7 @@ pki() {
 package() {
 	cd "$builddir"
 	make DESTDIR="$pkgdir" install || return 1
-	install -dm700 -o taskd -g taskd "$pkgdir"/var/lib/$pkgname
```

Comment:
This is correct. However, `pkgusers="taskd"` and `pkggroups="taskd"` must be defined in the abuild.